frontend/src/components/NotificationBox/NotificationBox.js

```diff
@@ -1,13 +1,16 @@
-import { useAuth } from "../Authentication/FirebaseAuth/FirebaseAuth";
+import React, {useState, useEffect} from 'react';
+import {useAuth} from "../Authentication/FirebaseAuth/FirebaseAuth";
 
 import styles from "./NotificationBox.module.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import "../Tabs/CommonPageStyling.css";
 
-import { Alert } from "react-bootstrap";
+import {Alert} from "react-bootstrap";
 import Notification from "../Notification/Notification";
-
+import axios from 'axios';
 
 function NotificationBox(props) {
-    let { currentUser } = useAuth();
+    let {currentUser} = useAuth();
     [
         'primary',
```

Comment:
A map returns the modified array. This doesn't seems to be used anywhere? Maybe I'm wrong. Could you explain?